test/profiling/allocations/AllocTestPublisher.hpp

```diff
@@ -34,9 +34,9 @@
 #include <fastdds/dds/topic/Topic.hpp>
 #include <fastdds/dds/topic/TypeSupport.hpp>
 
-#include "AllocTestType.h"
+#include "AllocTestType.hpp"
 
-class AllocTestPublisher : public eprosima::fastdds::dds::DataWriterListener
```

Comment:
This was like this on purpose, we'd like to remove the pattern of having a separate listener class, so when this test was created, it was the first to use the new approach.

Please revert this